Neptune: ps5 uses the BSD resolver/sockets and getaddrinfo like freebsd.

--- a/lib/libUPnP/CMakeLists.txt	2026-09-25 15:14:44.701927042 +0000
+++ b/lib/libUPnP/CMakeLists.txt	2026-09-25 15:14:44.706092709 +0000
@@ -96,7 +96,7 @@
                       Neptune/Source/System/Win32/NptWin32Time.cpp)
 endif()
 
-if(CORE_SYSTEM_NAME STREQUAL "freebsd")
+if(CORE_SYSTEM_NAME STREQUAL "freebsd" OR CORE_SYSTEM_NAME STREQUAL "ps5")
   list(APPEND SOURCES Neptune/Source/System/Bsd/NptBsdResolver.cpp
                       Neptune/Source/System/Bsd/NptBsdSockets.cpp)
 endif()
@@ -104,7 +104,7 @@
 
 add_library(upnp STATIC ${SOURCES} ${HEADERS})
 target_compile_definitions(upnp PUBLIC -DNPT_CONFIG_ENABLE_LOGGING)
-if(CORE_SYSTEM_NAME STREQUAL "freebsd")
+if(CORE_SYSTEM_NAME STREQUAL "freebsd" OR CORE_SYSTEM_NAME STREQUAL "ps5")
   target_compile_definitions(upnp PUBLIC -DNPT_CONFIG_HAVE_GETADDRINFO)
 endif()
 
